### PR TITLE
UI improvements pre tico

### PR DESCRIPTION
**Pull request by Bettelstab**
Addresses some of https://github.com/nimiq/safe/issues/61

### CODE REVIEW

File: src/elements/x-safe.js [hunk 1]
```diff
@@ -69,11 +70,21 @@ export default class XSafe extends MixinRedux(XElement) {
             <section class="content nimiq-dark content-width">
                 <nav class="actions floating-actions">
                     <div class="floating-btn">
-                        <button new-tx><span>Send</span></button>
+                        <button new-tx>
+                            <div>
+                                <svg xmlns="http://www.w3.org/2000/svg"><path d="M15,20H9V12H4.16L12,4.16L19.84,12H15V20Z" /></svg>
```
Comment (danimoh) [inline src/elements/x-safe.js]: why as `svg` now?
Also the arrows in Julian's design are thinner, which looks better

File: src/elements/x-safe.css [hunk 2]
```diff
@@ -341,20 +336,23 @@ nav.actions.floating-actions {
...
+.floating-btn button {
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
 }
 
-.floating-btn [new-tx]:disabled {
-    background-image: url('data:image/svg+xml,<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path fill="%234463D7" opacity="0.6" d="M15,20H9V12H4.16L12,4.16L19.84,12H15V20Z" /></svg>');
+.floating-btn button > div {
+    display: flex;
```
Comment (danimoh) [inline src/elements/x-safe.css]: I suggest keeping the icons as `background-image`, then they can be centered by `background-position` without the need for the additional div.

File: src/elements/x-safe.css [hunk 3]
```diff
@@ -389,11 +401,11 @@ nav.actions.floating-actions {
 
 @media (max-width: 690px) {
     .header-top {
-        height: 108px;
+        height: 10rem;
```
Comment (danimoh) [inline src/elements/x-safe.css]: the safe is mostly in px. Does it make sense to switch to rem here?

File: src/elements/x-safe.css [hunk 4]
```diff
@@ -439,15 +451,15 @@ nav.actions.floating-actions {
     }
 
     .floating-btn button {
+        justify-content: left;
```
Comment (danimoh) [inline src/elements/x-safe.css]: Looks broken for contacts button between breakpoints at 730px and 480px.

File: src/elements/x-safe.css [hunk 5]
```diff
@@ -460,6 +472,12 @@ nav.actions.floating-actions {
         font-family: inherit;
     }
 
+    .floating-btn button > div {
+        display: flex;
```
Comment (danimoh) [inline src/elements/x-safe.css]: `display: flex` already set in line 346

File: src/elements/x-safe.js (hunk 1 above)
Comment (Bettelstab) [reply src/elements/x-safe.js]: fill color can be applied that way.

File: src/elements/x-safe.css (hunk 2 above)
Comment (Bettelstab) [reply src/elements/x-safe.css]: see above

File: src/elements/x-safe.css (hunk 4 above)
Comment (Bettelstab) [reply src/elements/x-safe.css]: ... but this button is removed anyway

File: src/elements/x-safe.css (hunk 2 above)
Comment (danimoh) [reply src/elements/x-safe.css]: Changed it back to `background-image`

**Pull request merged**